bin/fast-build

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Builds CLI binary for current platform only and outside docker to speed up things. Suitable for local development.
+
+bindir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+. $bindir/_tag.sh
+
+CONDUIT_SKIP_CLI_CONTAINER=1 bin/docker-build
```

Comment:
Related to my comment above in the `docker-build` script, I'd rename this script `fast-build-cli-bin`, and I'd remove this line, since `docker-build` will now call this script.